include/votca/csg/molecule.h

```diff
@@ -70,14 +70,16 @@ class Molecule {
   std::string getBeadName(Index bead) { return _bead_names[bead]; }
 
   /// Add an interaction to the molecule
-  void AddInteraction(Interaction *ic) { _interactions.push_back(ic); }
+  void AddInteraction(const Interaction *ic) { _interactions.push_back(ic); }
```

Comment:
Who in general has the ownership of interaction topology/residues or what?

Reply:
Topology has ownership of all the topology objects (beads, residues, interactions, molecules).